CONTRIBUTING.md

```diff
@@ -20,6 +20,7 @@ information on using pull requests.
 Prerequisites:
 
 * You need to have Python installed.
```

Comment:
Do we want to merge these two lines given they both talk about Python runtime requirements?

Reply:
Makes sense. It's probably required to have all versions installed anyways regardless of whether you are only running tests.